grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/ArmeriaMessageDeframer.java

```diff
@@ -317,6 +317,7 @@ public void close() {
                 unprocessed.forEach(ByteBuf::release);
             } finally {
                 unprocessed = null;
+                closeWhenComplete = false;
```

Comment:
Is this necessary?

Reply:
Technically, we don't need it now. I added this line for integrity. It seems weird to me that both `isClosed()` and `isClosing() return true. 😅

Reply:
OK, SGTM.